### ISSUE TITLE
[Bug]: wrong translation of title and description xliff-Export

### ISSUE THREAD

**Issue by sara-luise** [pimcore/pimcore#13141]
### Expected behavior

When you want to have a translation from for example 'de' to 'en': 
at the end of the exported .xliff file (in XLIFF Export/Import) the `settings~-~title` and `settings~-~description` should be german in the source and english in the target.

....
```  
<trans-unit id="settings~-~title">
           <source xml:lang="de"><![CDATA[Hallo]]></source>
           <target xml:lang="en"><![CDATA[Hello]]></target>
</trans-unit>
<trans-unit id="settings~-~description">
           <source xml:lang="de"><![CDATA[Das ist eine Testnachricht]]></source>
           <target xml:lang="en"><![CDATA[This is a testmessage]]></target>
</trans-unit>
```
...

### Actual behavior

The source and target content of both are in the source language.

....
``` 
<trans-unit id="settings~-~title">
          <source xml:lang="de"><![CDATA[Hallo]]></source>
         <target xml:lang="en"><![CDATA[Hallo]]></target>
</trans-unit>
<trans-unit id="settings~-~description">
          <source xml:lang="de"><![CDATA[Das ist eine Testnachricht]]></source>
          <target xml:lang="en"><![CDATA[Das ist eine Testnachricht]]></target>
</trans-unit>
```
...

### Steps to reproduce

Mentioned above

**Comment by sara-luise** [comment]
Solution: 

In `Pimcore\Translation\ExportDataExtractorService\DataExtractor\DocumentDataExtractor\addSettings`:

...
``` 
foreach ($result->getTargetLanguages() as $targetLanguage) {
                if (isset($translations[$targetLanguage])) {
                    $targetDocument = Document::getById($translations[$targetLanguage]);

                    if ($targetDocument instanceof  Document\PageSnippet) {
                        $targetData['title'][$targetLanguage] = $document->getTitle();
                        $targetData['description'][$targetLanguage] = $document->getDescription();
                    }
                }
            }
```
...

-> change to
...
``` 
foreach ($result->getTargetLanguages() as $targetLanguage) {
                if (isset($translations[$targetLanguage])) {
                    $targetDocument = Document::getById($translations[$targetLanguage]);

                    if ($targetDocument instanceof  Document\PageSnippet) {
                        $targetData['title'][$targetLanguage] = $targetDocument->getTitle();
                        $targetData['description'][$targetLanguage] = $targetDocument->getDescription();
                    }
                }
            }
```
...

**Comment by dvesh3** [comment]
@sara-luise thanks for reporting the issue. maybe you could provide a PR since you already know the solution 😉 thanks!

**Comment by dvesh3** [comment]
Fixed by #13145
